### ISSUE TITLE
V1 parallel states

### ISSUE THREAD

**Issue by billsworld** [angular-ui/ui-router#2837]
Trying to get parallel states working with V1, removing the incompatible ui-router-extras sticky states dependency if possible. The use case is a dialog that can open on top of multiple other states. The dialog should change the url/state but it should keep the below state intact. It should run in parallel to the below state so that the below state is still visible and would not have to be reloaded when the dialog is closed. Here is a simple plunker showing as close as I am able to get by returning false in the `onEnter` callback http://plnkr.co/edit/wAtuqUf3KEL3CNRUt9gb?p=preview This is not ideal because the url never changes.

Is there any way to get this so that it will update the url when the dialog is opened as well? What is going to be the proper way to get this sort of thing working with V1?

**Comment by christopherthielen** [comment]
Hi @billsworld 
Sticky states will need to be rewritten to support ui-router 1.0.0

The `$view` service should enable a much simpler sticky states implementation to be written.  I'm going to spend some energy on that after ui-router 1.0 is nearer to final release.

**Comment by billsworld** [comment]
Thanks @christopherthielen for the update. We are looking forward to it! Let me know if there is anything I can do to help out along the way.

**Comment by david-hollifield** [comment]
+1

**Comment by ssidorchick** [comment]
+1 would be nice to have this feature since cannot use 1.0 without it

**Comment by christopherthielen** [comment]
+1 also would like this feature :)

**Comment by christopherthielen** [comment]
If anyone wants to help, I have a plan for implementing these in 1.0.  I would be happy to provide guidance for a PR while I'm busy working on 1.0 core.

**Comment by ssidorchick** [comment]
Probably I can take a look. @christopherthielen could you provide some basic steps how it can be started?

**Comment by NeoGenet1c** [comment]
Happy to help you guys with that as well.

**Comment by christopherthielen** [comment]
Note: the parallel states/sticky states project now lives at https://github.com/ui-router/sticky-states
